src/rsz/test/report_equiv_cells.ok

```diff
@@ -0,0 +1,160 @@
+[INFO ODB-0227] LEF file: sky130hd/sky130hd.tlef, created 13 layers, 25 vias
+[INFO ODB-0227] LEF file: sky130hd/sky130hd_std_cell.lef, created 437 library cells
+[INFO ODB-0128] Design: shorted_outputs
+[INFO ODB-0130]     Created 3 pins.
+[INFO ODB-0131]     Created 3 components and 14 component-terminals.
+[INFO ODB-0132]     Created 2 special nets and 0 connections.
+[INFO ODB-0133]     Created 4 nets and 6 connections.
+The following 29 cells are equivalent to sky130_fd_sc_hd__buf_1:
+=============================================================================
+          Cell                              Area   Area Leakage  Leakage  VT
+                                           (um^2)  Ratio  (W)     Ratio  Type
+=============================================================================
+sky130_fd_sc_hd__clkbuf_1                   3.754  1.00 1.18e-12  1.00     0
+sky130_fd_sc_hd__buf_2                      5.005  1.33 3.93e-12  3.33     0
+sky130_fd_sc_hd__clkbuf_2                   5.005  1.33 4.15e-12  3.52     0
+sky130_fd_sc_hd__clkbuf_4                   7.507  2.00 4.60e-12  3.90     0
```

Comment:
could the VT type get a human readable name?

Reply:
Its possible to have multiple implants for a cell so the name might become rather long.  Is that ok?

Reply:
I would think something is better than 0, 1, 2, 3, so I think a long string would be acceptable

Reply:
Will be addressed in a follow on PR